src/toM3u8.js

```diff
@@ -1,3 +1,52 @@
+// Object => Object[]
+const values = o => Object.keys(o).map(k => o[k]);
```

Comment:
surprised we don't have the babel setting for this turned on. Maybe move this to `utils/object.js?`

Reply:
Babel will only transform syntax and not polyfill APIs. I'll move this to `object.js`

Reply:
yeah, I meant that we don't include the Object.values built-in https://github.com/babel/babel/blob/master/packages/babel-preset-env/data/built-in-features.js#L185

Reply:
We should investigate the runtime transform, particularly with babel 7. It allows us to inline polyfill these things without it changing the world.